metals/src/main/scala/scala/meta/internal/debug/protocol/LaunchParameters.scala

```diff
@@ -0,0 +1,7 @@
+package scala.meta.internal.debug.protocol
+
+final case class LaunchParameters(
```

Comment:
Can we please remove the nested `debug.protocol` package? By convention in this codebase, we use a flat structure of packages where each package has a functional role

```
package scala.meta.internal.metals // application code for language server
package scala.meta.internal.mtags  // library code for Scala and Java source indexing
package scala.meta.internal.pc     // library code for using presentation compiler
package scala.meta.internal.builds // library code for dealing with sbt/Gradle/Bloop builds
package scala.meta.internal.debug  // library code for dealing with Debug Adapter Protocol
```

When creating a new class, it should either have a unique name in an existing package or go into a new toplevel `internal.$NAME` package. The reason I dislike deeper nesting within those packages is that it makes it easy to end up with conflicting names that are difficult to distinguish, for example `LaunchParameters` in this PR

<img width="597" alt="Screenshot 2019-05-20 at 13 12 45" src="https://user-images.githubusercontent.com/1408093/58018035-07a64900-7b02-11e9-94b7-0fbcc199b63e.png">

I admit, that the `internal.metals` package has too much code at the moment and would benefit from breaking it into smaller packages. I think @tgodzik already did a great job of extracting the build related code into `internal.builds` and I think we could do the same for other related functionality like indexing/search.